tests/test_for_suite/test_for_cli/test_for_runner.py

```diff
@@ -93,3 +137,69 @@ def test_run_skip_unchanged_true(runner, mock_repo):
     number_of_result_files = len(os.listdir(join(mock_repo[0], 'results')))
     os.chdir(previous_wd)
     assert number_of_result_files == 3
+
+
+def test_new(runner, tmpdir):
+    """Expect memote new to create a cookiecutter repo."""
+    target_dir = str(tmpdir)
+    user_responses = "John\nj@d.com\nJD\nmock-repo\nmock-repo\n" \
+                     "description\n2019-02-07\n2019\n0.1.0\n" \
+                     "default\ndefault\ngh-pages"
+    result = runner.invoke(cli, [
+        "new", "--directory", target_dir], input=user_responses)
+    assert result.exit_code == 0
+
+
+@pytest.mark.skip(reason="Currently fails with: fatal: could not read "
+                         "Username for 'https://github.com': Device not "
+                         "configured. In addition, incorporating github "
+                         "credentials is a bit of an issue.")
+def test_online(runner, mock_repo, monkeypatch):
```

Comment:
If the idea is that this function should be tested manually from time to time, it could be helpful to document how to do that.

Reply:
I'm hoping to be able to redefine 'origin' to point to another local repo. This way we could run this test routinely and this specific documentation may be superfluous.

Reply:
I'm afraid you cannot have a situation where Travis will be able to connect to a local repo :( so I'm indeed also unsure when and how to run this test.

Reply:
Any interaction with TravisCI (or GitHub) is being mocked in this test since we don't want to test API interactions. Only the final actions performed by `memote online` are essentially tested here:
- Loading the travis.yml
- Storing the token in the travis.yml
- Adding, committing and pushing to origin

Even though this isn't much I think it still warrants thorough test coverage.

Reply:
Oh, yes of course. That means you just extract the compressed repository to two different locations and declare one to be the origin for the other.

Maybe it's time we create a 'slow' pytest mark and only run slow tests on Travis rather than locally in development.

Reply:
Opened #633 for this purpose! https://github.com/opencobra/memote/issues/633